Dockerfile

```diff
@@ -5,7 +5,10 @@ COPY ./web/package.json ./
 COPY ./web/package-lock.json ./
 RUN npm ci --silent
 COPY ./web ./
-RUN npm run build
+
+ARG REACT_APP_POKE_API
```

Comment:
I think this needs to change from ARG to ENV as ARGS are only available during the build phase and with env we can change it during execution. If we need it before building. Then I think we need to use a similar approach that we have for the analytics config where the BE prints the config to the FE from the `index.html` file